app/views/practices/_adoption_form.html.erb

```diff
@@ -0,0 +1,176 @@
+      <label class='usa-radio__label' for='<%= "status_in_progress#{adoption&.diffusion_history&.id}" %>'>In progress
+        adoption</label>
+    </div>
+    <div class="usa-radio position-relative">
+      <%= radio_button(nil, 'status', 'Completed', class: 'usa-radio__input adoptions-radio-button', required: true, checked: status == complete ? 'checked' : false, id: "status_completed#{adoption&.diffusion_history&.id}") %>
+      <label class='usa-radio__label' for='<%= "status_completed#{adoption&.diffusion_history&.id}" %>'>Completed
+        adoption</label>
+    </div>
+    <div class="usa-radio position-relative">
+      <%= radio_button(nil, 'status', 'Unsuccessful', class: 'usa-radio__input adoptions-radio-button', required: true, checked: status == unsuccessful ? 'checked' : false, id: "status_unsuccessful#{adoption&.diffusion_history&.id}") %>
+      <label class='usa-radio__label' for='<%= "status_unsuccessful#{adoption&.diffusion_history&.id}" %>'>Unsuccessful
+        adoption</label>
+    </div>
+  </div>
+
+  <div class="margin-bottom-3">
+    <p class="margin-bottom-1">
+      <span class="fw-bold">Adopting facility:&nbsp;</span>
+      Select the state and facility where this practice is being adopted.</p>
+    <div class="grid-row grid-gap-4">
+      <div>
+        <%
+          state_select_selector = adoption ? "editor_state_select_#{adoption.id}" : 'editor_state_select'
+          facility_select_selector = adoption ? "editor_facility_select_#{adoption.id}" : 'editor_facility_select'
+          form_id = "adoption_form#{adoption&.diffusion_history&.id}"
+        %>
+
+        <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+            // enable/disable date adoption finished select dropdowns
+          var radios = document.querySelectorAll('form[id="<%= form_id %>"] input[type=radio][name="status"]');
+
+          function changeHandler(event) {
+            var end_month_el = document.getElementById('date_ended_month<%= adoption&.diffusion_history&.id %>');
+            var end_year_el = document.getElementById('date_ended_year<%= adoption&.diffusion_history&.id %>');
+            if (this.value === 'In progress') {
+              end_month_el.classList.add('dm-color-disabled');
+              end_month_el.disabled = true;
+              end_year_el.classList.add('dm-color-disabled');
+              end_year_el.disabled = true;
+            } else {
+              end_month_el.classList.remove('dm-color-disabled');
+              end_month_el.disabled = false;
+              end_year_el.classList.remove('dm-color-disabled');
+              end_year_el.disabled = false;
+            }
+          }
+
+          Array.prototype.forEach.call(radios, function(radio) {
+            radio.addEventListener('change', changeHandler);
+          });
+        <% end %>
+        <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
```

Comment:
do we need this `javascript_tag`?

Reply:
can probably combine, yeah

Reply:
oh, or get rid of. must've missed it in my refactoring :smile: